### DISCUSSION TITLE
Let's put out a tutorial for beginners.

### DISCUSSION THREAD

**Discussion by zhaoxiaozhang** [TongchengOpenSource/smart-doc#389] [Q&A]
I strongly suggest creating a tutorial for beginners, as some really don't know how to do it.
强烈建议出个小白教程吧，有些真不知道怎么弄啊

**Comment by chenqi146** [comment]
Are you referring to examples of Spring Boot single-module and multi-module projects?
是指springboot单模块和多模块的示例吗
